lib/mix/tasks/firmware.burn.ex

```diff
@@ -75,6 +75,8 @@ defmodule Mix.Tasks.Firmware.Burn do
            ask_pass = System.get_env("SUDO_ASKPASS") || "/usr/bin/ssh-askpass"
            System.put_env("SUDO_ASKPASS", ask_pass)
            {"sudo", ["fwup"] ++ args}
+        {_, :nt} ->
+           {"fwup", args}
```

Comment:
It has been working great!  I cross compiled it from a Linux VM, and haven't had a single issue.  It even detects the SD Card correctly :+1: